backend/src/test/java/com/bakdata/conquery/integration/json/QueryTest.java

```diff
@@ -59,8 +86,97 @@ public void importRequiredData(StandaloneSupport support) throws IOException, JS
 
 		IntegrationUtils.importTableContents(support, Arrays.asList(content.getTables()), support.getDataset());
 		support.waitUntilWorkDone();
+		importIdMapping(support);
+		importPreviousQueries(support);
+	}
+
+	public void importIdMapping(StandaloneSupport support) throws JSONException, IOException {
```

Comment:
Die Funktionen sehen aus wie Util methoden. Können die nicht in die LoadingUtil?